packages/create-pantheon-decoupled-kit/src/templates/next-wp/.env.example

```diff
@@ -1,7 +1,21 @@
-# Optional environment variables are commented out
-WPGRAPHQL_URL=
-IMAGE_DOMAIN=
-#PANTHEON_UPLOAD_PATH=/example
-#PREVIEW_SECRET=
-#WP_APPLICATION_USERNAME=
-#WP_APPLICATION_PASSWORD=
+# Copy as .env.development.local to override envars for local development
+
+# The graphql endpoint of your WordPress instance
+WPGRAPHQL_URL=https://my-backend-site.pantheonsite.io/wp/graphql
+
+# The secret set on your WordPress Preview Site. Required for preview functionality
+PREVIEW_SECRET=super-secret-preview-secret
+
+# The domain where your images are hosted. By default, we will set the domain of the WPGRAPHQL_URL
+# to the value of IMAGE_DOMAIN. Set this if your images are hosted somewhere other than your WordPress instance
+# See https://nextjs.org/docs/pages/api-reference/components/image#domains for more information
+IMAGE_DOMAIN=some-other-domain.example
```

Comment:
I think the convention used previously was good

```
# Optional environment variables are commented out
```

Having this variable be not commented out makes me think I need it. And if someone does blindly copy the whole file, and start changing values, they'll end up with broken images as long as they keep `IMAGE_DOMAIN=some-other-domain.example` right?